docs/dev_guide/ceph_s3_tests/ceph_s3_tests_pending_list_status.md

```diff
@@ -31,34 +31,7 @@ Attached a table with tests that where investigated and their status (this table
-| test_alias                                                | Not Implemented |                      |                      |
-| test_complex_expressions                                  | Not Implemented |                      |                      |
-| test_datetime                                             | Not Implemented |                      |                      |
-| test_alias_cyclic_refernce                                | Not Implemented |                      |                      |
-| test_csv_parser                                           | Not Implemented |                      |                      |
-| test_csv_definition                                       | Not Implemented |                      |                      |
-| test_true_false_datetime                                  | Not Implemented |                      |                      |
-| test_schema_definition                                    | Not Implemented |                      |                      |
-| test_when_then_else_expressions                           | Not Implemented |                      |                      |
-| test_coalesce_expressions                                 | Not Implemented |                      |                      |
-| test_trim_expressions                                     | Not Implemented |                      |                      |
-| test_cast_expressions                                     | Not Implemented |                      |                      |
-| test_version                                              | Not Implemented |                      |                      |
-| test_truefalse_trim_expressions                           | Not Implemented |                      |                      |
-| test_escape_expressions                                   | Not Implemented |                      |                      |
-| test_bool_cast_expressions                                | Not Implemented |                      |                      |
-| test_case_value_expressions                               | Not Implemented |                      |                      |
-| test_progress_expressions                                 | Not Implemented |                      |                      |
-| test_output_serial_expressions                            | Not Implemented |                      |                      |
\ No newline at end of file
+| test_generate_projection                                  | Faulty Test     |                      |                      |
+| test_alias_cyclic_refernce                                | Faulty Test     |                      |                      |
+| test_schema_definition                                    | Faulty Test     |                      |                      |
+| test_progress_expressions                                 | Faulty Test     |                      |                      |
```

Comment:
Would you please add the links to the file?
This way we would have all the details we need to understand why the tests are faulty.

Reply:
I don't follow. What file?

Reply:
This file (the file where I added this comment): [docs/dev_guide/ceph_s3_tests/ceph_s3_tests_pending_list_status.md](https://github.com/noobaa/noobaa-core/pull/7222/files/9e5658ee92d32b4e5978465d1e3729958af31a29#diff-7cd72e38295c1d1f096ebd7dd5d8ae6da87ee21c2fe5cbc78101ded33f50feee).
Fill `Issue Number` column with attached link and (if needed) `Additional Comments`, for example, line 11 in this file.